File: ProyectoCsharp/Modelo/Area.cs
public class Area{
	private long idArea;
	private string nombreArea;

    //Constructores
    public Area(){
        idArea = 0;
		nombreArea = null;
	}
	public Area(long idArea, string nombreArea){
        IdArea = idArea;
        NombreArea = nombreArea;
	}
	
	~Area(){
		nombreArea = null;
	}

    //Set y get

    public long IdArea { get => idArea; set => idArea = value; }
    public string NombreArea { get => nombreArea; set => nombreArea = value; }

    public override string ToString()
    {
        return NombreArea;
    }
}

File: TrabajoJavaLp2/src/modelo/Area.java
package modelo;
public class Area{
	private long idArea;
	private String nombreArea;

    

	public Area(){
            idArea = 0;
            nombreArea = null;
	}
	public Area(long idArea, String nombreArea){
            this.idArea = idArea;
            this.nombreArea = nombreArea;
	}
        
        public long getIdArea() {
            return idArea;
        }

        public void setIdArea(long idArea) {
            this.idArea = idArea;
        }

        public String getNombreArea() {
            return nombreArea;
        }

        public void setNombreArea(String nombreArea) {
            this.nombreArea = nombreArea;
        }
}